docs/source/NEWS.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 20.1.2 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- fix ``ModuleNotFoundError: No module named 'cStringIO'`` in ldaptor-ldap2pdns.
+- move scripts to console_scripts entry_points
```

Comment:
```suggestion
- move scripts to console_scripts entry_points. The ``ldaptor.scripts`` package
  has been included in the standard library on a provisional basis. Backwards
  incompatible changes (up to and including removal of the module) may occur
  if deemed necessary.
```

Reply:
or maybe it would be better in `ldaptor._scripts` as I want to change the `def main(` to be `async def main(reactor):` for `task.react`

Reply:
well... I think that each script should have an high level entry point which will return a deferred.

and then we can have a thin wrapper which runs the deferred in reactor and stops the reactor.
This can be done via t.i.task.react https://twistedmatrix.com/documents/current/api/twisted.internet.task.html#react

Reply:
Yeah I've got a PR to add corofn support to t.i.task.react

Reply:
I think it's best to have it in `ldaptor._scripts` until we've nailed down a nice public interface